src/platforms/flutter/troubleshooting.mdx

```diff
@@ -33,3 +33,7 @@ The configuration for symbolication of Native events (C/C++) is documented in ou
 ## Native Symbolication on iOS/macOS
 
 Flutter `split-debug-info` and `obfuscate` flags are supported on iOS/macOS. They require compiling your app using Flutter, version `3.7.0` and above and the Sentry Flutter SDK, version `6.10.0` and above.
+
+## Source Context
+
+Source Context support requires compiling your app using the `split-debug-info` build parameter and the latest Flutter (`master` channel), you must also upload [debug symbols](/platforms/flutter/upload-debug/).
```

Comment:
Is `Source Context support` something that's obvious to Flutter devs? Is this something we can link to?

Reply:
I don't think we have a product's page for source context but if you don't know what that means, a [blogpost example](https://blog.sentry.io/2017/11/08/one-year-of-elixir/#source-code-context).